modules/Package.swift

```diff
@@ -193,10 +209,7 @@ let package = Package(
         ),
         .target(
             name: "Generated",
-            resources: [.process("Resources")],
-            plugins: [
-                .plugin(name: "SwiftGenPlugin", package: "SwiftGenPlugin")
-            ]
+            resources: [.process("Resources")]
```

Comment:
As unfortunate as this is, the SwiftGen plugin is, as of the time of this writing, broken.